tests/benchmarks/implicit_edge_deletion.yml

```diff
@@ -4,6 +4,7 @@ remote:
   - type: oss-standalone
 dbconfig:
   - init_commands:
+      - '"GRAPH.QUERY" "g" "CREATE INDEX FOR (n:N) ON (n.v)"'
       - '"GRAPH.QUERY" "g" "UNWIND range(0, 100000) AS x CREATE (src:N {v: x}), (src)-[:R]->(:N), (src)-[:R]->(:N), (src)-[:R]->(:N)"'
```

Comment:
Each iteration on the `UNWIND` loop creates 4 nodes, all of the same Label `N` but only 1/4 has a `v` attribute.
I think it would better to change this:
```
UNWIND range(0, 100000) AS x CREATE (src:N {v: x}), (src)-[:R]->(), (src)-[:R]->(:A), (src)-[:R]->(:B)"
```